agent/lm_agent/reconciliation.py

```diff
@@ -38,91 +32,81 @@
     scontrol_update_reservation,
 )
 
-RECONCILE_URL_PATH = "/lm/api/v1/license/reconcile"
 
-
-def get_greatest_grace_time(job_id: str, grace_times: Dict[int, int], booking_rows: List) -> int:
+def get_greatest_grace_time_for_job(grace_times: Dict[int, int], job_bookings: List[BookingSchema]) -> int:
     """
-    Find the greatest grace_time for the given job_id.
+    Find the greatest grace_time for each feature booked by the given job_id.
```

Comment:
I think the word `each` here is used wrongly because the function is returning the greatest grace time value between all features, not the greatest for each feature

Reply:
Fixed it, thanks!